content/en/docs/concepts/package-configuration.md

```diff
@@ -0,0 +1,616 @@
+```
+
+---
+
+Verify the configuration by running `curl` on the deployment pod.
+
+   Example:
+
+   ```bash
+   kubectl exec -i nginx-webroot-59cb6c745d-5f2b8 -- curl -s localhost
+   ```
+
+   Example Output:
+
+   ```bash
+   <html>
+     <body>
+       <h1>hi there</h1>
+     </body>
+   ```
+
+## Examples
+
+Let’s examine a couple more examples to illustrate how package configuration works.
+
+### Example 1: Customizing Image Versions
+
+Suppose you have multiple applications with different frontend and
+backend images and ports. You want users to easily customize the
+images and ports based on the application. Here's a step-by-step process:
+
+1. **Create a PackageManifest** \
+Define the scope and phases of your package in your `manifest.yaml`:
+
+   ```yaml
+   apiVersion: manifests.package-operator.run/v1alpha1
+   kind: PackageManifest
+   metadata:
+     name: frontend-backend
+   spec:
+     scopes:
+     - Namespaced
+     phases:
+     - name: deploy
+     availabilityProbes:
+     - probes:
+       - condition:
+           type: Available
+           status: "True"
+       - fieldsEqual:
+           fieldA: .status.updatedReplicas
+           fieldB: .status.replicas
+       selector:
+         kind:
+           group: apps
+           kind: Deployment
+   ```
+
+1. **Define Package Configuration** \
+Under the `config` section, add the configuration schema for your
+package to your `manifest.yaml`. Allow users define image and port versions.
+
+   ```yaml
+   config:
+     openAPIV3Schema:
+     properties:
+       frontendImage:
+          description: frontendImage is the reference to the image containing the frontend application.
+          type: string
+       databaseImage:
+          description: backendImage is the reference to the image containing the backend application.
+          type: string
+       frontendPort:
+          description: Port for the frontend container.
+          type: integer
+          default: 80
+       databasePort:
+          description: Port for the database container.
+          type: integer
+          default: 5432
+     type: object
+   ```
+
+1. **Add the following to the template section of your `deployment.yaml.gotmpl`:**
+
+   ```yaml
+    template:
+      metadata:
+        labels:
+          app.kubernetes.io/name: frontend-backend
+          app.kubernetes.io/instance: "{{.package.metadata.name}}"
+      spec:
+        containers:
+          - name: frontend
+            image: "{{.config.frontendImage}}"
+            ports:
+              - containerPort: {{.config.frontendPort}}
+
+          - name: database
+            image: "{{.config.databaseImage}}"
+            ports:
+              - containerPort: {{.config.databasePort}}
+   ```
+
+1. **Build and push your package to your image repository:**
+
+   ```yaml
+   kubectl package validate <your-template-directory>
+   ```
+
+   ```yaml
+   kubectl package build -t <your-image-url-goes-here> --push <your-template-directory>
```

Comment:
Technically speaking a URL or URI must include a scheme.
The more correct term in this case is image reference as in `<your-image-reference-goes-here>`.

Reply:
I agree - I was trying to keep it consistent with the other docs. I've changed it here, but it  should probably be updated in the rest of the documentation as well.